collections/_article/dev-snapshot-godot-4-2-beta-1.md

```diff
@@ -0,0 +1,346 @@
+
+A dedicated shoutout to [Rakka Rage](https://github.com/rakkarage) who has been fixing up the tilemap editor since Godot 4.1, and [Thiago Lages de Alencar](https://github.com/thiagola92) who fixed their first bug in this release. They are among many more people making the engine better one contribution at a time ([GH-77316](https://github.com/godotengine/godot/pull/77316), [GH-79678](https://github.com/godotengine/godot/pull/79678), [GH-80943](https://github.com/godotengine/godot/pull/80943), [GH-80968](https://github.com/godotengine/godot/pull/80968)).
+
+
+### Editor and usability
+
+With every Godot release there are always a few hundred changes aimed at improving user experience and fixing usability issues, changing editor tools to better suit most common needs and to get in the way as little as possible.
+
+With many changes to cover, it's hard to pick the most impactful ones, because it's the sum of all the work that ensures you are happy to open the editor every day. The next few subsections try their best to highlight in broad strokes all the amazing work being done on the editor.
+
+But before we go into more specific changes in various editor areas, an honorable mention goes to [MewPurPur](https://github.com/MewPurPur) who has been tirelessly working on adding missing icons and optimizing existing ones ([GH-78858](https://github.com/godotengine/godot/pull/78858), [GH-78903](https://github.com/godotengine/godot/pull/78903), [GH-79431](https://github.com/godotengine/godot/pull/79431), [GH-80102](https://github.com/godotengine/godot/pull/80102), [GH-80103](https://github.com/godotengine/godot/pull/80103), [GH-80113](https://github.com/godotengine/godot/pull/80113), [GH-80129](https://github.com/godotengine/godot/pull/80129), and many more).
+
+#### Code editor
+
+We can start with the code editor, which now supports code regions (for GDScript), allowing you to break up scripts into named blocks, foldable and easy to navigate, without affecting the flow of the program ([GH-74843](https://github.com/godotengine/godot/pull/74843) by [Jean-Michel Bernard](https://github.com/jmb462)). The logic behind commenting out parts of your code also becomes more predictable with improvements to the toggle comments behavior by [Michał Iwańczuk](https://github.com/iwek7) ([GH-44557](https://github.com/godotengine/godot/pull/44557)).
```

Comment:
I'll drop the "can" but keep the general structure. I'm also going to break it up differently, because the last clause relates to the first part of the sentence, not to the "foldable and easy to navigate" part.